.github/actions/kind/action.yml

```diff
@@ -35,10 +35,14 @@ runs:
         EOF'
 
     - name: Setup KinD cluster
-      uses: helm/kind-action@v1.8.0
+      uses: helm/kind-action@v1
       with:
         cluster_name: cluster
-        version: v0.17.0
+        # The kind version to use
+        version: v0.25.0
+        # The Docker image for the cluster nodes - https://hub.docker.com/r/kindest/node/
+        node_image: kindest/node:v1.30.6@sha256:b6d08db72079ba5ae1f4a88a09025c0a904af3b52387643c285442afb05ab994
```

Comment:
we will hit rate limiting issues if we pull from docker, is there an alternative?

Reply:
You sure? all upstream GHA jobs pull from there.

Reply:
+1, high possibility of rate limit issues, would suggest using quay images instead.

Reply:
The current image is `kindest/node:v1.25.3`
```
 • Ensuring node image (kindest/node:v1.25.3) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.25.3) 🖼
```
This change only make it clear what image we are going to use that in this case is: `kindest/node:v1.30.6`

A rate limit can happen with `kindest/node:v1.25.3` or with `kindest/node:v1.30.6`

Reply:
> You sure? all upstream GHA jobs pull from there.

No, I'm not sure! If this is the case it works for me.